app/models/ni_number.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+class NiNumber
+  include ActiveModel::Model
+
+  attr_accessor :trn_request_id
+
+  validates :ni_number,
+            format: {
+              with: /\A[a-z]{2}[0-9]{6}[a-d]{1}\Z/i,
+              message: 'Enter a National Insurance number in the correct format',
```

Comment:
Maybe this? (untested code, maybe needs `activerecord.` prefix)

```suggestion
              message: I18n.t('errors.models.ni_number.attributes.ni_number.blank')
```

I think not using `I18n.t` in the tests makes sense, but what about model valdiations?

Reply:
I think I can simply remove this line from the model and it will use the locales file.

Reply:
Good call.